.github/workflows/ts-tests.yml

```diff
@@ -38,8 +38,7 @@ jobs:
         if: ${{ runner.os == 'macOS' || runner.os == 'Linux' }}
       - name: Perform TypeScript tests
         run: |
-          ./gradlew targetTest -Ptarget=TypeScript
-        # -Druntime="git://github.com/lf-lang/reactor-ts.git#master"
+          ./gradlew targetTest -Ptarget=TypeScript -Druntime="git://github.com/lf-lang/reactor-ts.git#dnet-ad-hoc"
```

Comment:
Can we just update the submodule instead?  Let's get the TS changes merged first and remove this change.

Reply:
Since the PR [reactor-ts](https://github.com/lf-lang/reactor-ts/pull/296) has been merged, I made the GitHub action use the `master` branch in [83bd58f](https://github.com/lf-lang/lingua-franca/pull/2400/commits/83bd58f7eeb5a8a0e5234ab299186f6a9ebf16b0).